### PR TITLE
feat(.gov): add level 4 lab 203

### PR DESCRIPTION
**Pull request by krab7191**
Closes https://github.com/ibm-build-lab/vest-content/issues/142

Content changes:

- Add Level 4 subfolder with basic lab files
- Add 203 lab & screenshots

Code changes:

- Any code changes here

### CODE REVIEW

**Comment by krab7191** [comment]
NOTE: I felt there were too many repetitive steps and so compressed a lot. Waiting for my new TZ reservation to finish so I can get new screenshots.

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+## Configuring the governance console
+
+1. ### Turn on OpenPages integration
+    1. Make sure you're logged in to Cloud Pak for Data using the *Console Route*, *Username*, and *Password* like in the user creation step above.
+    2. From the home screen, click the **hamburger menu** in the top left, then under **AI governance** click **External models**. <br />![external models](../images/203/watsonxgov-l4_external-models.webp)
+    3. On the **External models** page, click the blue **Setup external model governance** button. <br />![setup external gov](../images/203/watsonxgov-l4_setup-external-gov.webp)
+    4. Click the toggle button for **External model governance** to activate it. <br />![external model gov](../images/203/watsonxgov-l4_external-model-gov.webp)
+    5. Click on the **Inventory** dropdown and select the **Platform assets catalog** option.
+    6. Click on the **Default owner** dropdown and select the **admin** user, then click Apply. You will return to the previous screen.
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:34]: `Apply` -> `**Apply**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    7. Click the toggle button for **Governance console (IBM OpenPages) integration** to activate it. <br />![openpages switch](../images/203/watsonxgov-l4_openpages-switch.webp)
+    8. On the **Sync with Governance console (IBM OpenPages)** page, click on the **OpenPages location** dropdown and select IBM Cloud Pak for Data. Note that, depending on the version you have, this option may be called watsonx instead. <br />![openpages integration](../images/203/watsonxgov-l4_openpages-integration.webp)
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:36]: `IBM Cloud Pak for Data` -> `**IBM Cloud Pak for Data**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    4. In the **Add Object Type Access** panel, scroll down until you can see several Risk options. Check the boxes for **Risk**, **Risk Assessment**, **Risk Assessment Eval**, and **Risk Eval**. <br />![role templates risk](../images/203/watsonxgov-l4_role-templates-risk.webp)
+    5. Keep scrolling down and check the box for **Use Case Review** as well.
+    6. Click the blue **Add** button at the bottom of the panel to add access. Note that you have only added Read level access at this point. The Add Object Type Access panel closes, and Risk as well as the other objects now appear in the main access list.
+    7. Back in the main access list, select the options you just added: **Risk**, **Risk Assessment**, **Risk Assessment Eval**, **Risk Eval**, and **Use Case Review**.
+    8. At the top of the table, click the **Edit** button, <br />![role templates edit](../images/203/watsonxgov-l4_role-templates-edit.webp) and then change the **Write**, **Delete**, and **Associate** permissions to **Granted**. <br />![role templates granted](../images/203/watsonxgov-l4_role-templates-granted.webp) Click Done. You have completed configurations for the governance console.
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:101]: `Click Done` -> `Click **Done**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+    The datamart requires either a Db2 or PostgreSQL database.
+
+1. ### Create a database
+    1. Log in to Cloud Pak for Data as the admin user in your environment, using the Cloud Pak for Data *Console Route* and *password* gathered in step 8 of the Environment setup part of [Lab 201](/watsonx/watsonxgov/level-4/201).
+    2. From the home screen, click the hamburger menu in the upper left, then under **Data**, click **Databases**. ![create db](../images/203/watsonxgov-l4_create-db.webp) <br />The reservation system may have created a database for you, in which case it will appear here. You may skip ahead to [Gather database credentials](#gather-database-credentials). If you do not see a database, you will need to follow the steps below to create one.
+    3. Click the **Create a database** button, and verify that the **Db2** tile is selected as the database type, with the most recent software version selected in the dropdown. Click Next to continue. <br /> ![create db2](../images/203/watsonxgov-l4_create-db2.webp)
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:110]: `Click Next` -> `Click **Next**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+    4. Under **Storage option**, make sure **Single location for all data** is selected, and click next. ![create db configure](../images/203/watsonxgov-l4_create-db-configure.webp) Keep the defaults and move on to the credentials page.
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:111]: `Click Next` -> `Click **Next**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+    5. Under credentials, use the **Input method** dropdown to select **Generate a Kubernetes secret** and click next. <br />![create db creds](../images/203/watsonxgov-l4_create-db-creds.webp)
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:112]: `Click Next` -> `Click **Next**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+    6. For Storage, select **Create new storage**, **Use storage template**, and in the dropdown select **ocs-storagecluster-cephfs**. Reduce the size to **50 GiB** and click next. <br /> ![create db storage](../images/203/watsonxgov-l4_create-db-storage.webp)
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:113]: `Click Next` -> `Click **Next**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+    7. On the Finalize screen, you can safely ignore the warning message. Set the Display name to `cpd-database` and click Create. The database will take roughly 30 minutes to create. When it is finished, you can proceed to the next step.
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:114]: `click Create` -> `click **Create**`
`30 minutes` -> `**30 minutes**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+    2. Locate the **Deployment id** in the details towards the bottom of the page. Select and save the value somewhere for easy reference later. <br /> ![db id](../images/203/watsonxgov-l4_db-id.webp)
+    3. Add a prefix of `c-` and a suffix of `-db2u` to the deployment ID value from the previous step to create the database host name. For example, in the screenshot above, the database has a deployment ID of **db2oltp-1719228613082150**. The corresponding host name would be **c-db2oltp-1719228613082150-db2u**.
+
+3. ### Configure the monitoring service
+
+    You have now created a database and are ready to connect it to the watsonx.governance monitoring service.
+
+    1. From the CP4D home page, click the hamburger menu in the top left, and under **Services**, click **Instances**.
+    2. Locate the **IBM Watson OpenScale instance** from the list. Note that, on IBM Cloud, this service has been renamed to watsonx.governance. Future software releases will likely incorporate this change as well, so this screen may change to reflect the new name.
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:129]: Make the Note a `> NOTE`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+    5. From the **System setup** screen, click the Pencil icon on the Database tile to edit database information.
+    6. Use the following values for the fields:
+        - Database type: **db2**
+        - Database location: **Other**
+        - Hostname or IP Address: **The value you constructed earlier ending in -db2u**
+        - Database: `BLUDB`
+        - Username: `admin`
+        - Password: **Your Cloud Pak for Data admin password** <br /> ![openscale config](../images/203/watsonxgov-l4_openscale-config.webp)
+    7. Click Connect. The monitoring service will attempt to connect to the database using the credentials you supplied. If the connection fails, double-check that you have constructed the hostname correctly, and that you are using the admin username and password that you use to log into the Cloud Pak for Data or watsonx home screen.
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:141]: `Click Connect` -> `Click **Connect**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+    8. After successfully connecting, set **Schema** to **Auto-create a new schema**, and click save. When the changes have successfully saved, the watsonx.governance monitoring service will be configured and operational. <br /> ![openscale db connect](../images/203/watsonxgov-l4_openscale-db-connect.webp)
+
+4. ### Add the user to the monitoring service
+
+    In order to perform model evaluations, the user you created in previous steps needs to be granted access to the monitoring service.
+
+    1. From the CP4D home screen use the hamburger menu, then **services** -> **Instances**
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:148]: `**services**` -> `**Services**`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+    2. Locate the OpenScale service in the table and open the ellipsis menu, then click on **Manage access**. <br /> ![openscale manage access](../images/203/watsonxgov-l4_openscale-manage-access.webp)
+    3. Click on **Add users**, then select the user you created, and set the role to Admin, then click add. <br /> ![openscale user add](../images/203/watsonxgov-l4_openscale-user-add.webp) <br />The created user will now have Admin privileges, which will allow them to add machine learning providers. Note that assigning Editor privileges would allow the user to create and remove model subscriptions and configure models for monitoring, but not add the deployment spaces or third-party environments that host the models.
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:150]: `Admin` -> `**Admin**`
`click add` -> `click **add**`

**Review by MaxShapiro32 — CHANGES_REQUESTED** [review]

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+
+This lab depends on the setup in [Lab 201](/watsonx/watsonxgov/level-4/201), which means having a running CP4D environment from TechZone.
+
+## Create a user
+
+In this step, you will create a sample user in Cloud Pak for Data. This step is required for the rest of the hands-on labs.
+
+1. First log in to Cloud Pak for Data using the *Console Route*, *Username*, and *Password* you got from the *update-configmap-success* CP4D pipeline logs in [Lab 201](/watsonx/watsonxgov/level-4/201).
+2. From the home screen, click the **hamburger menu** in the top left, then under **Administration**, click **Access control**. <br />![access control](../images/203/watsonxgov-l4_access-control.webp)
+3. Click the blue **Add users** button, and then fill out the profile information. Make sure not to forget your username or password! Click **Next**. <br />![user details](../images/203/watsonxgov-l4_user-details.webp)
```
[the remark this answers is not in the document]
Comment (krab7191) [reply content/Watsonx/WatsonxGov/Level-4/203.md:21]: Ah ok. Will fix up

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+
+4. ### Add the user to the monitoring service
+
+    In order to perform model evaluations, the user you created in previous steps needs to be granted access to the monitoring service.
+
+    1. From the CP4D home screen use the hamburger menu, then **Services** -> **Instances**
+    2. Locate the OpenScale service in the table and open the ellipsis menu, then click on **Manage access**. <br /> ![openscale manage access](../images/203/watsonxgov-l4_openscale-manage-access.webp)
+    3. Click on **Add users**, then select the user you created, and set the role to **Admin**, then click **add**. <br /> ![openscale user add](../images/203/watsonxgov-l4_openscale-user-add.webp) <br />The created user will now have Admin privileges, which will allow them to add machine learning providers. Note that assigning Editor privileges would allow the user to create and remove model subscriptions and configure models for monitoring, but not add the deployment spaces or third-party environments that host the models.
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:151]: This screenshot should be updated to show the `complianceofficer` username to be consistent

`then select the user you created` can specify `complianceofficer`

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 15]
```diff
@@ -0,0 +1,155 @@
...
+
+This lab depends on the setup in [Lab 201](/watsonx/watsonxgov/level-4/201), which means having a running CP4D environment from TechZone.
+
+## Create a user
+
+In this step, you will create a sample user in Cloud Pak for Data. This step is required for the rest of the hands-on labs.
+
+1. First log in to Cloud Pak for Data using the *Console Route*, *Username*, and *Password* you got from the *update-configmap-success* CP4D pipeline logs in [Lab 201](/watsonx/watsonxgov/level-4/201).
+2. From the home screen, click the **hamburger menu** in the top left, then under **Administration**, click **Access control**. <br />![access control](../images/203/watsonxgov-l4_access-control.webp)
+3. Click the blue **Add users** button, and then fill out the profile information. Use `complianceofficer` for the username, as this will be referenced in future labs. Make sure not to forget your password! Click **Next**. <br />![user details](../images/203/watsonxgov-l4_user-details.webp)
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:21]: This screenshot should be updated to show the `complianceofficer` username to be consistent

File: content/Watsonx/WatsonxGov/Level-4/203.md [hunk 16]
```diff
@@ -0,0 +1,155 @@
...
+    10. Click the **Inventories** menu item on the left, and then click the **New inventory** button. <br />![new inventory](../images/203/watsonxgov-l4_new-inventory.webp)
+    11. In the name field, enter `High Oaks Banks Model Use Cases`. Ensure that the **Add collaborators after creation** box is checked, and then click **Create**.
+    12. On the **Set collaborators** page, click **Add collaborators**, then **Add user**, and start typing the user ID of the user you created to find it. Make sure **Admin** is selected, then click add. <br />![add collaborator](../images/203/watsonxgov-l4_add-collaborator.webp)<br />![collaborators](../images/203/watsonxgov-l4_collaborators.webp)<br />Now that you have configured Cloud Pak for Data to use the watsonx governance console (OpenPages), model use cases created in the governance console will appear in the inventory in Cloud Pak for Data, and vice versa. You can close the **Set collaborators** and **Manage** windows to return to the **AI use case** screen, then return to the Cloud Pak for Data home page by clicking the title in the upper left corner.<br /><br />
```
Comment (MaxShapiro32) [inline content/Watsonx/WatsonxGov/Level-4/203.md:40]: These screenshots should be updated to show the `complianceofficer` username to be consistent

`start typing the user ID of the user you created` can specify `complianceofficer`

**Review by MaxShapiro32 — APPROVED** [review]

**Pull request merged**